### DISCUSSION TITLE
FBA or FBN ?

### DISCUSSION THREAD

**Discussion by tetsuoo-online** [Rparadise-Team/Koriki#17] [General]
So, I'm confused about a few things :
- there are 2 folders FBA and FBN but it seems that Final Burn Neo is the newest version of FinalBurn Alpha
- I have kof96 and metalSlug X that will work only in FBNeo, is FBAlpha even needed ?
- It seems that all the themes have a FBA background picture but nothing for FBN. I'm using ATC Elegant Color theme, is there a blank/template PNG with transparency so that I can customize a new FBN picture ? That would help a lot, thanks

Edit: I managed to add a new path in theme.ini for FBN. Well, that's a start :)

**Comment by eduardofilo** [comment]
1. FBN is the replacement for FBA, indeed. But FBA is more widely used and there are many people who have romsets for that emulator, that's why it has its own standalone system set up in SimpleMenu. With FBN we did something similar, that is to say we prepared a system in SimpleMenu inside the Arcade section configured to include romsets from this system as you can see inside `.simplemenu/section_groups/arcades.ini`:

```
[FINALBURN NEO]
execs = /mnt/SDCARD/.simplemenu/launchers/fbneo_libretro
romDirs = /mnt/SDCARD/Roms/FBN/
romExts = .zip
aliasFile = /mnt/SDCARD/.simplemenu/arcade.txt
```

But in the end, as we had to generate covers for all the themes, not only for the ATC collection made by our colleague @albgr1979, we ended up not doing it.

2. The FBN core is configured in the MAME system, so I recommend that you install those ROMs in the ARCADE directory and then do a core setup for them. Here you can read about how to do it: https://github.com/Rparadise-Team/Koriki/wiki/En:-04.-Problem-solving#emulator-selection

3. I will pass the question to @albgr1979  who is the creator of the ATC themes.

**Reply by albgr1979** [reply]
What exactly is needed? The background of FINAL BURN NEO?

**Reply by tetsuoo-online** [reply]
> What exactly is needed? The background of FINAL BURN NEO?

Yes please ^^ If you don't have the time I could make use of a blank template to customize it

**Reply by albgr1979** [reply]
I didn't plan to do it because of what @eduardofilo has explained to you, but tell me what theme you want me to do it on and I'll send it to you.

**Comment by albgr1979** [comment]
@tetsuoo-online Here you have it. The only thing you have to do is unzip the content of the .zip file in the root of the micro-SD card.

[Koriki - ATC Elegant Color (FBN).zip](https://github.com/Rparadise-Team/Koriki/files/10754025/Koriki.-.ATC.Elegant.Color.FBN.zip)
